### PR TITLE
feat(versions): Add current version to version history

### PR DESCRIPTION
**Pull request by jstoffan**
Current Version

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jstoffan has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/versions/VersionsSidebarContainer.js [hunk 1]
```diff
@@ -57,19 +55,38 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         this.props.onVersionChange();
     }
 
-    handleFetchVersionsError = ({ message }) => {
+    fetchFile = () => {
```
Comment (jeremypress) [inline src/elements/content-sidebar/versions/VersionsSidebarContainer.js]: flow types?

**Pull request merged**